src/app/components/core/configuration-manager/configuration-group/configuration-group.component.html

```diff
@@ -9,8 +9,8 @@
     </ul>
   </nav>
   <div class="configuration_group_wrapper">
-    <nav id="group_navigation" class="groupNav dragscroll mouse-scroll" role="tablist">
-      <div id="groupNavContents" class="nav" [ngClass]="{'groupNav_contents': !sourceName}">
+    <nav [id]="'group_navigation_'+from + '_' + sourceName" class="groupNav dragscroll mouse-scroll" role="tablist">
```

Comment:
In next line ID convention is camelCase and here snake_case?

Reply:
ID convention updated